Dockerfile

```
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ARG BASE=golang:1.16-alpine3.14
+ARG BASE=golang:1.16-alpine3.12
```

Moving target here, sorry. We are now on Go 1.17 and alpine 3.15
```suggestion
ARG BASE=golang:1.17-alpine3.15
```